doc/reference/tooling/tcm/tcm_access_control.rst

```diff
@@ -0,0 +1,383 @@
+can apply to your organization.
+
+You can edit the |tcm| password policy on the **Password policy** page.
+There are the following password policy settings:
+
+-   **Minimal password length**.
+-   **Do not use last N passwords**.
+-   **Password expiration in days**. After this number of days, the user loses access
+    to any objects and functions except password change. After they change the password,
+    all the access rights are returned.
+-   **Password expiration warning in days**. After this number of days, the user
+    sees a warning that they must change their password.
+-   **Block after N login attempts**.
+-   **User lockout time in seconds**.
+-   **Password must include**. Characters and symbols that must be present in passwords:
+
+    -   **Lowercase characters (a-z)**
+    -   **Uppercase characters (A-Z)**
+    -   **Digits (0-9)**
+    -   **Symbols (such as !@#$%^&\*()_+№"':,.;=][{}`?>/.)**
+
+.. _tcm_access_control_sessions:
+
+Sessions
+--------
+
+Administrators can view and revoke user sessions in |tcm|. All active sessions
+are listed on the **Sessions** page. To revoke a session, click **Revoke** in the
+**Actions** menu of the corresponding table row.
+
+To revoke all sessions of a user, go to **Users** and click **Revoke all sessions**
+in the **Actions** menu of the corresponding table row.
+
+.. _tcm_access_control_permissions_ref:
+
+Permissions reference
+---------------------
+
+.. _tcm_access_control_permissions_admin:
+
+Administrative permissions
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The following administrative permissions are available in |tcm|:
+
+..  list-table::
+    :widths: 30 70
+    :header-rows: 1
+
+    *   -   Permission
+        -   Description
+
+    *   -   ``admin.clusters.read``
+        -   View connected clusters' details
+
+    *   -   ``admin.clusters.write``
+        -   Edit cluster details and add new clusters
+
+    *   -   ``admin.users.read``
+        -   View users' details
+
+    *   -   ``admin.users.write``
+        -   Edit user details and add new users
+
+    *   -   ``admin.roles.read``
+        -   View roles' details
+
+    *   -   ``admin.roles.write``
+        -   Edit roles and add new roles
+
+    *   -   ``admin.addons.read``
+        -   View add-ons
+
+    *   -   ``admin.addons.write``
+        -   Edit add-on flags
+
+    *   -   ``admin.addons.upload``
+        -   Upload new add-ons
+
+    *   -   ``admin.auditlog.read``
+        -   View audit log configuration and read audit log in |tcm|
+
+    *   -   ``admin.auditlog.write``
+        -   Edit audit log configuration
+
+    *   -   ``admin.sessions.read``
+        -   View users' sessions
+
+    *   -   ``admin.sessions.write``
+        -   Revoke users' sessions
+
+    *   -   ``admin.ldap.read``
+        -   View LDAP configurations
+
+    *   -   ``admin.ldap.write``
+        -   Manage LDAP configurations
+
+    *   -   ``admin.passwordpolicy.read``
+        -   View password policy
```

Comment:
Looks like the first column should be a bit wider:
<img width="848" alt="image" src="https://github.com/tarantool/doc/assets/38073144/8926dae7-0d83-466a-be3c-0202ba6c288e">

Reply:
Fixed